### ISSUE TITLE
`browser.alarms.getAll()` returns `undefined` instead of array

### ISSUE THREAD

**Issue by ghost** [webexts/webextensions-api-fake#2]
[`browser.alarms.getAll()`](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/alarms/getAll) should return an Array. However, it is returning `undefined` using v0.7.5.

```javascript
const browserFake = require("webextensions-api-fake");
global.browser = browserFake();

(async () => {
    const alarms = await browser.alarms.getAll();
    console.log(alarms);
    // Should return `[]` but returns `undefined`
})();
```

**Comment by ghost** [comment]
Probably caused by a dependency. I filed a related issue acvetkov/sinon-chrome#90

**Comment by stoically** [comment]
`browser.alarms` is currently not implemented. You can see the list of working fakes [`here`](https://github.com/webexts/webextensions-api-fake#api-fake). I'll look into implementing `alarms`.

`sinon-chrome` just provides sinon stubs for all APIs and doesn't actually fake behavior, you need to define the desired behavior yourself. E.g. for `alarms.getAll` you could do

```js
browser.alarms.getAll.callsFake(async () => []);
```

or

```js
browser.alarms.getAll.resolves([]);
```

**Comment by stoically** [comment]
Published 0.8.0 with the alarms fake.

If you don't want to wait for the timeout or interval you could use [sinon fake-timers](https://sinonjs.org/releases/latest/fake-timers/).
